meta-ibm/recipes-phosphor/chassis/mux-workarounds/mux-workarounds.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# everest system only
+if [ $(fw_printenv fitconfig | grep everest) ]; then
+    gpiopath="/sys/bus/platform/drivers/gpio-keys-polled"
+    drvpath="/sys/bus/i2c/drivers/pca954x"
+
+    # remove gpio dependencies
+    echo gpio-keys-polled > $gpiopath/unbind
+
+    # bus 4
+    if [ -e $drvpath/4-0070 ]; then echo 4-0070 > $drvpath/unbind; fi
+    i2cset -f -y 0 0x61 7 0x15 b
```

Comment:
Can we please put these `i2cset` calls in a named function? Otherwise the intent isn't clear and the code's a bit impenetrable. Applies to the remaining `i2cset` calls below.

Reply:
Sure